src/openforms/prefill/api/urls.py

```diff
@@ -1,8 +1,29 @@
 from django.urls import path
 
-from .views import PluginAttributesListView, PluginListView
+from .views import (
+    PluginAttributesListView,
+    PluginListView,
+    PluginObjectsAPIAttributesListView,
+    PluginObjectsAPIObjecttypeListView,
+    PluginObjectsAPIObjecttypeVersionListView,
```

Comment:
Please define these views in their plugin package `openforms.prefill.contrib.object_api.api.views`. We should *not* pollute the plugin-agnostic python code/modules with plugin-specific features.